bittide-instances/exe/post-vex-riscv-tcp-test/Main.hs

```diff
@@ -48,15 +67,15 @@ case_testGdbProgram :: Assertion
 case_testGdbProgram = do
```

Comment:
I have 2 remarks on things you didn't touch in this PR, but since you do touch the files you can fix them here. Please update the test documentation in `bittide-instances/exe/post-vex-riscv-tcp-test/Main.hs`, which you copy-pasted in #578. Also rename the test case to `case_testTcpProgram` or something like that.

Reply:
Fixed in #613